datadog_checks_base/tests/test_agent_check.py

```diff
@@ -3,10 +3,13 @@
 # (C) Datadog, Inc. 2018
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+import sys
+
 import mock
 import pytest
 from six import PY3
 
+from datadog_checks.base.stubs import datadog_agent
```

Comment:
Not actually used in here ?

Reply:
Good catch, yeah not since we started using the stub and could actually do mocking :)